ddtrace/contrib/pytest/plugin.py

```diff
@@ -62,7 +62,7 @@ def encode_test_parameter(parameter):
 
 def is_enabled(config):
     """Check if the ddtrace plugin is enabled."""
-    return config.getoption("ddtrace") or config.getini("ddtrace")
+    return (config.getoption("ddtrace") or config.getini("ddtrace")) and not config.getoption("no-ddtrace")
```

Comment:
sorry to do it to you, but would you be willing to make this it's own PR.

we probably want to add a release note as a feature here for `--no-ddtrace` since we are modifying the pytest plugin that customers also use.